deployments/liqo/subcharts/authService/templates/auth-service-deploy.yaml

```diff
@@ -195,11 +202,50 @@ metadata:
   name: auth-service
   namespace: {{ .Release.Namespace }}
 spec:
+  {{ if .Values.ingress.enable }}
+  type: ClusterIP
+  {{ else }}
   type: NodePort
+  {{ end }}
   selector:
     run: auth-service
   ports:
     - name: https
       protocol: TCP
-      port: 443
+      {{ if .Values.ingress.enable }}
+      port: 5000
       targetPort: 5000
+      {{ else }}
+      port: 443
+      targetPort: 443
+      {{ end }}
+---
+{{ if .Values.ingress.enable }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  labels:
+    run: auth-service
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+  name: auth-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - http:
+        paths:
+          - backend:
+              service:
+                name: auth-service
+                port:
+                  number: 5000
+            path: /
+            pathType: Prefix
+      {{ if (.Values.ingress.host) }}
+      host: {{ .Values.ingress.host }}
+      {{ end }}
+  tls:
+    - hosts:
+      - {{ .Values.ingress.host }}
```

Comment:
host is not protected by the same if present in http field.

Reply:
thanks, now it is protected